### PR TITLE
Add unzip to kas-base

### PR DESCRIPTION
**Pull request by HannahKiekens**
Required for bb recipes that have zip files in SRC_URI

### CODE REVIEW

**Comment by HannahKiekens** [comment]
Zip files in SRC_URI worked fine with the 3.0.2 version of kas-container
This adds back the missing dependency

**Comment by HannahKiekens** [comment]
Fixes:

DEBUG: Executing python function do_unpack
NOTE: Unpacking **REDACTED** to /build/tmp/work/cip-core-bullseye-amd64/**REDACTED**
/bin/sh: 1: unzip: not found
ERROR: Unpack failure for URL: '**REDACTED**'. Unpack command PATH="/work/isar/scripts:/work/isar/bitbake/bin:/usr/sbin:/usr/bin:/sbin:/bin" unzip -q -o '/build/downloads/**REDACTED**' failed with return value 127
DEBUG: Python function do_unpack finished

**Comment by henning-schild** [comment]
This is about "kas-isar" right? because we have it in "kas", where you want to drop it when moving it to "kas-base"

It would also be nice to see this being done via the mailing list workflow.

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN apt-get update && \
     localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
     apt-get install --no-install-recommends -y \
         python3-pip python3-setuptools python3-wheel python3-yaml python3-distro python3-jsonschema python3-newt \
-        gosu lsb-release file vim less procps tree tar bzip2 zstd pigz lz4 tmux libncurses-dev \
+        gosu lsb-release file vim less procps tree tar bzip2 zstd pigz lz4 unzip tmux libncurses-dev \
```
Comment (henning-schild) [inline Dockerfile:13]: when we have it here we do no longer need it in the "kas" container for yocto, final container of that Dockerfile

**Comment by henning-schild** [comment]
Signed-Off seems missing

**Comment by henning-schild** [comment]
aka https://groups.google.com/g/kas-devel/c/vjYH8oYL-cE

**Pull request closed without merging**